### ISSUE TITLE
Training DeeplabV3+ using the generated pseudo labels

### ISSUE THREAD

**Issue by Billy-ZTB** [ISCSS-FangTeam/ACGC#3]
Hello! I've generated pseudo labels according to the code you provided, and pseudo label iou on **training** set is similar to that reported in your papaer(0.5% iou lower than yours on vaihingen and potsdam dataset). However, after training with these pseudo labels, iou on both test sets are more than 3% lower than yours. Could you please tell me how the hyper parameters are set when training deeplabv3+ model? Also I wonder if the pseudo mask metrics reported in your papar is on training set, and deeplabv3+ results on test set.

**Comment by labiao** [comment]
Thank you for your attention. For clarity, the metrics based on pseudo masks are reported on the training set, while the DeepLabv3+ metrics are reported on the test set. Checkpoints were selected by the peak building-class IoU, not the peak mIoU. Additionally, we observed that the PaddleSeg implementation of DeepLabv3+ exhibits test-set performance differences across hardware setups. I hope this addresses your concern.
Please also check out our more recent work: https://github.com/labiao/BAC-AR-Net.git

**Comment by labiao** [comment]
[ind_AdvCAM.yml](https://github.com/user-attachments/files/22917825/ind_AdvCAM.yml)

<img width="664" height="40" alt="Image" src="https://github.com/user-attachments/assets/8b3a89fc-5ad0-42da-8b6d-eb9ce23b3f49" />

**Comment by Billy-ZTB** [comment]
Thanks! I'll check on that later.
